metricshub-web/react/src/components/status/otel-status-icon.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Box, CircularProgress, Tooltip } from "@mui/material";
+import { useAppSelector } from "../../hooks/store";
+import otelColor from "../../assets/opentelemetry-icon-color.png";
+
+/**
+ * OpenTelemetry status icon component
+ * @param {object} props - Component props
+ * @param {object} props.sx - MUI sx prop for styling
+ * @returns JSX.Element
+ */
+export default function OtelStatusIcon({ sx = {} }) {
```

Comment:
Also, there are 3 statuses for the OpenTelemetry Collector:

* Enabled AND running (okay, color icon)
* Enabled but NOT running (danger, add a small red cross to clearly show there is a problem)
* Disabled (okay, greyed out icon) (this happens when MetricsHub is configured not to automatically launch the OpenTelemetry Collector) (wait... @NassimBtk do you confirm we can disable the automatic spawn of the Otel collector, right?)